graph-builder/src/graph.rs

```diff
@@ -170,6 +195,7 @@ pub fn run<'a>(settings: &'a config::AppSettings, state: &State) -> ! {
             }
             Err(err) => {
                 UPSTREAM_ERRORS.inc();
+                UPSTREAM_ERRORS_SUBSEQUENT.inc();
```

Comment:
Prometheus can derive this from the rate of increase of `UPSTREAM_ERRORS`, no need to have a separate metrics for it (and it is not used for anything else in our own logic).